classes/class06/twoter/routes/main.js

```diff
@@ -33,11 +33,7 @@ app.get('/', function(req, res, next) {
 
 		// Check to see if user can see the page.
 		if (req.session.authorName == null) {
-			res.render('main', {
-				isAuthenticated: req.isAuthenticated(),
-				user: req.user,
-				authorLoggedIn: false
-			});
+			res.redirect("/login");
 		} else {
 		  res.render('main', {
 		  	isAuthenticated: req.isAuthenticated(),
```

Comment:
You're using `req.isAuthenticated()` to tell your front end if the user is logged in, but using `req.session.authorName == null` to verify on the server. Try to keep that consistent in case the two get out of sync somehow.